javascript/javascript1/week3/note-taking-app.js

```diff
@@ -0,0 +1,74 @@
+console.log("------>>> Save a Note <<<------");
+
+const notes = [];
+
+function saveNote(content, id) {
+  notes.push({ content, id });
+}
+
+saveNote(
+  "Employee Engagement: a monthly 'Lunch & Learn' program where team members can share their expertise on various topics",
+  1
+);
+saveNote(
+  "A new project management tool, like Asana, to improve task tracking and team communication",
+  2
+);
```

Comment:
I will suggest running Formating with ESLint on this file, it will make it easier to read. 
Then Move all const (Because their value never changes ) to top of your file. 

It will make reading this file much easier. Other than that looks correct :)